features/step_definitions/hooks-js.js

```diff
@@ -0,0 +1,42 @@
+const { spawn } = require('child_process');
+const { expect } = require('chai');
+const { Given, When, Then } = require('cucumber');
+
```

Comment:
I want the step definitions to be like LEGO pieces one day, but I don't want to LEGO-ize them prematurely. I thought I'd start with this, where each step definition is very specific to the feature file in hand, and that we'd LEGO-ize and DRY **when needed** once we're adding more feature files.